projects/packages/calypsoify/src/js/mods-gutenberg.js

```diff
@@ -1,44 +1,19 @@
-/* eslint-disable no-var */
-/* global wp, calypsoifyGutenberg */
+import '../scss/style-gutenberg.scss';
+import { dispatch, select } from '@wordpress/data';
 
-jQuery( function ( $ ) {
-	// Force fullscreen mode for iframed post editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-post' ) &&
-		! wp.data.select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
-	}
+// Force fullscreen mode for iframed post editor.
+if (
+	select( 'core/edit-post' ) &&
+	! select( 'core/edit-post' ).isFeatureActive( 'fullscreenMode' )
+) {
+	dispatch( 'core/edit-post' ).toggleFeature( 'fullscreenMode' );
+}
 
-	// Force fullscreen mode for iframed site editor.
-	if (
-		wp &&
-		wp.data &&
-		wp.data.select( 'core/edit-site' ) &&
-		! wp.data.select( 'core/edit-site' ).isFeatureActive( 'fullscreenMode' )
-	) {
-		wp.data.dispatch( 'core/edit-site' ).toggleFeature( 'fullscreenMode' );
-	}
-
-	var editPostHeaderInception = setInterval( function () {
-		var $closeButton = $( '.edit-post-fullscreen-mode-close__toolbar a' );
-		if ( $closeButton.length < 1 ) {
-			return;
-		}
-		clearInterval( editPostHeaderInception );
-
-		if ( calypsoifyGutenberg.closeUrl ) {
-			$closeButton.attr( 'href', calypsoifyGutenberg.closeUrl );
-			$closeButton.attr( 'target', '_parent' );
-		}
-	} );
-
-	$( 'body.revision-php a' ).each( function () {
-		var href = $( this ).attr( 'href' );
+document.addEventListener( 'DOMContentLoaded', function () {
```

Comment:
I wonder whether it'd be better to use [`@wordpress/dom-ready`](https://www.npmjs.com/package/@wordpress/dom-ready) instead.

Reply:
I've now removed all of this. I've worked on a separate diff to make sure the admin-plugins version of Calypsoify works properly instead.